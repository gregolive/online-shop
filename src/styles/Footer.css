footer {
  color: var(--text);
  background: var(--secondary);
  border-top: 1px solid black;
  height: 3rem;
  display: flex;
  align-items: center;
  justify-content: center;
}

footer a {
  color: var(--text);
}

footer a:hover {
  text-decoration: underline;
}