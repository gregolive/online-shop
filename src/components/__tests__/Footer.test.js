import { render, screen } from '@testing-library/react';
import Footer from '../Footer';

test('renders github link', () => {
  render(<Footer />);
  expect(screen.getByRole('link').textContent).toMatch(/gregolive/);
});

it('renders consistently', () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
});
