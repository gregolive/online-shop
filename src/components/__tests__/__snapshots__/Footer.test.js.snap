// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders consistently 1`] = `
<div>
  <footer>
    <p>
      Â© 
      <a
        href="https://github.com/gregolive"
        rel="noreferrer"
        target="_blank"
      >
        gregolive
      </a>
    </p>
  </footer>
</div>
`;
